### PR TITLE
virsh: add network activation example

### PR DESCRIPTION
**Pull request by Managor**
- [ ] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ ] The page(s) have at most 8 examples.
- [ ] The page description(s) have links to documentation or a homepage.
- [ ] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ ] The page(s) follow the [style guide](/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Review by nelsonfigueroa — APPROVED** [review]

**Review by gutjuri — APPROVED** [review]

**Comment by Managor** [comment]
@principis You originally made this page. Do you think configuration dumping is the command to replace?

**Comment by principis** [comment]
> @principis You originally made this page. Do you think configuration dumping is the command to replace?

Sure, seems fine.

File: pages/common/virsh.md [hunk 1]
```diff
@@ -8,13 +8,13 @@
 
 `virsh connect {{qemu:///system}}`
 
-- List all domains:
+- Activate a network named `default`:
 
-`virsh list --all`
+`virsh net-start {{default}}`
```
Comment (spageektti) [inline pages/common/virsh.md:13]:
```suggestion
`virsh net-start default`
```

**Review by spageektti — CHANGES_REQUESTED** [review]

File: pages/common/virsh.md (hunk 1 above)
Comment (Managor) [reply pages/common/virsh.md:13]: The device name can change if you've created more than one.

Comment (spageektti) [reply pages/common/virsh.md:13]: The example description states that the name is "default," and we do not use placeholders in these cases.

Comment (Managor) [reply pages/common/virsh.md:13]: I've done that multiple times in the past. This is the first time anyone's taken note of it.

Comment (gutjuri) [reply pages/common/virsh.md:13]: I also think that it would be clearer to have a placeholder in this case.

Comment (spageektti) [reply pages/common/virsh.md:13]: There were some PRs removing placeholders in similar cases, for example, https://github.com/tldr-pages/tldr/pull/13723

Comment (Managor) [reply pages/common/virsh.md:13]: I really disagree with these changes. For me placeholders are there to indicate that an argument can be changed.

Comment (spageektti) [reply pages/common/virsh.md:13]: I don't have a strong opinion either way, so it is fine for me to include the placeholder here.

**Review by spageektti — APPROVED** [review]

**Pull request merged**